app/routes.py

```diff
@@ -1 +1,224 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db 
```

Comment:
One small suggestion here. As our projects start to scale and we are working with more and more models, it's a good idea to separate out our routes into individual files. This means we could have a task_routes.py file and a goal_routes.py file just to organize our file structure just a little further!